content/en/docs/concepts/services-networking/service.md

```diff
@@ -450,9 +450,8 @@ variables and DNS.
 ### Environment variables
 
 When a Pod is run on a Node, the kubelet adds a set of environment variables
-for each active Service.  It supports both [Docker links
-compatible](https://docs.docker.com/userguide/dockerlinks/) variables (see [makeLinkVariables](https://github.com/kubernetes/kubernetes/blob/dd2d12f6dc0e654c15d5db57a5f9f6ba61192726/pkg/kubelet/envvars/envvars.go#L72))
-and simpler `{SVCNAME}_SERVICE_HOST` and `{SVCNAME}_SERVICE_PORT` variables,
```

Comment:
I would hyperlink to https://docs.docker.com/network/links/ and say that the kubelet supports variables that are compatible with Docker Engine's legacy feature, _container links_.

We should not omit the mention of compatibility with that legacy feature as the behavior is still present (the removal of dockershim does not have any impact here).

Reply:
That makes sense! I've brought back the container links reference back to the content but moved it second in place after we have already mentioned the base `SERVICE_HOST` and `SERVICE_PORT` variables. Should we also add a comment in the link example below to demarcate the container link compatible env vars?